src/promptflow-core/promptflow/contracts/flow.py

```diff
@@ -702,12 +702,13 @@ def _update_working_dir(cls, working_dir: Path):
         sys.path.insert(0, str(working_dir))
 
     @classmethod
-    def from_yaml(cls, flow_file: Path, working_dir=None) -> "Flow":
+    def from_yaml(cls, flow_file: Path, working_dir=None, fallback_name=None) -> "Flow":
```

Comment:
fallback_name is not a good name.
Maybe just `name` and declare the priorities here

Reply:
Fixed.